.travis.yml

```diff
@@ -1,29 +1,28 @@
-- ./cd/deploy.sh
+    - $HOME/.m2/repository
+    - $HOME/.gradle
 
-env:
-  global:
-  - secure: trTkQFeh7MNfbk7jDeghRtqzizaWhs21dfxsA1eIxLlYRITdVdl/AeBXDslaNqQJq7FoqiZzDf23tQ0trZrNNTgxApZeSUMNQcCdWWgP6b/UA8lbbrWBykwI3cofqrDEUmPi7YLkApgZwCiyNebAOFmEDOGFGaQeQeJHy0Q+0vX8zp8iZ7aPEYfAKoASNv5SePxogBjWbmWihe7SQsP6hMkPcpYvzWG8MD7C8LG37HnpAO3uCDXmMr6wnQYEbObcuKhGtqrxGGfdlIxeesw7Imgqcu/xgEOOpmh/eo173yaMZHUlXeiHavjld/g+Ei1O5m+DdbP7ajVnRsvgCJ3NMIMoMlHVnlhrcbBW6isa6Zm58NktiFCF+RRf+7Ep47bqVfULzOfsN0w7Dmt/AkUNHm8vzIv4UB0TDnWQRBN/UIgj9Uh8HJXPcltlTBq2TQUlXTvFLDr2+xu4zou7VrpPKnHMVlsbsu6h3917quAJS4IDQjuuPOje/w4FSoil9gOMk685SaEyVoFOIwIq5of5ailUWlNY2Q6E3IsKiP0+PESnq31r9uxm7PDfMciS26aaoejHNlc42xrZsi/mbNxkIIO2p27+J3k81OiaHsyyIzmPtb23E6FyH83UY0TbLa3uScfdkaAeuOkU4sQKjK7kjhFHIXN3ItXPxrDJr/d5gX8=
-  - secure: fjB2MNht++6e/ZpRRVohbZBDzvqaI0KhHRNTq3f++CgIZioA1tazIRYog4G+u2sUCyl52KDpyjg2AjONccTFuoy/iHxKAHEnCgOLEvXDyxjHaZdF2SNVwvEmONiUEAKdvOeU+AWExE5N0SIajzzIc7U3ZHIPDCneaVuSykkzOTxckDxS5/QHt6eWuqJptnuyJyS/5Pmrd8r192sebOFsswKpM3yiHu4qf7TT1Sy8QQD3jrmVum+R4ofdYT3qBXZoXZcDwj41JK1zvOfw+D4Qnd7LXM9D2GAoIK/sas+/tORI/vbCEta/EFmVSrCfOtJBEpyLzcgsL5kq1vkJCM86GV2wre529gR3YyXEzKvoE2aL7DgS3mmM8ZLwAfR1SuBcf3H5a6xhFGb9WkQeMqF7hB3qQm34wkOAb0AgSRoKFsf81s3j+3nzMs+31X2CDFm3UQanALxHsIiUTiFVYo+cvKvkdQ4wPJgcufKcn/aGtOFhAGgTtRoVMu6wb3mKx28rkW6NrUQDc5dusE8miZoRApdQdilr7QTVDN+PW7JgoOEipizktbBGwKQEFwymnv87Vo/DbdFok5vEiFSL4hfBmAbODd2Yp6KxhApDdG4n+tyMG4VUOhUnUDAFOFyNINNuPuvFqQKm28WPHCLJkq7YFtmwvkcwjrLeM0DIGh+Tuq8=
-  - secure: tuIp+SQMx05Ls+Clt9oGFmZTerePQOZxy+DOb+v0fS/HY2DnXzTJhQ/ojAxinItH3b41xUZr3B12XwTOT+gq1rQ1A+KrZNswfzTVsyRFBMK/8KZYd6sXdax1RxBt3EUAYy78QPfJM0E6IY77rHdaTctgIrPl3REzyIfg8snsKxbawMdy6PScI/IZrz32QY50QDwBZXOSwvAvu7z9QAtriaNnT4LXxStcdN8inqO8Xkv8bXD9mUCEu2CI8PxIh96/guIU3FjY1CxTnmGFAVp6vylizEcz/3gwkM4CvztNxx5O3lmGPNjY1Pz0ZgVU4Bn08dys7KbvBpNRpspnbqxvXIju/bzkE7KbLW15GqeIstalEfG+r2ndgTg8AwZKeRm+4bDwtMsQshUn8Ym7jx69k7HvR6ySgbveNqUlmRfoJUsEW/RW/i/QDdwILTqDw/VsNXgJKCYmDMnJgbMpGuqN1RJvrg0VRC91GwL/HNh4KezQ4xeQ9WiTt/Q71rCCx0k7Aq1dp8M08xF+buu3rAyZECCKtK0QkuSk4X7N0WUwbFFBZeKeIzPc/DlbrUUJfyhaTkWlu+EBiEj8qIKCghcaaXZL2HVMfDK2MOK9mUgLSWx18cOoOmh6VKEwzPn+1aONkPVivuuzLwQQvESB/RJIJtv1aVU3I0E7uSu55c5/SfY=
-  - secure: MB8I+BHVP9QMaJQF1ALbp+/n0rfU1EVaPPOthn4MRoy6AUTL5Aigh/qJ5zghBR1yOOkbREJhgNkaN9Jwk8Oncxpl34FYj2+Sx9YPh2SB1q04k2VBeoP4lcn26ZMyGQJRjGSm37jwF/N2pEya7auYOo5EGwVFCGPndnA4YngsGiOfnlYWboDNp37RhqXcME6epo0pd/QYuJSGkjd5pMdEpenWdVmXkAxRLxFhgeKi7oI8sMZ8pR5I/i5jDlPHGlk2t9u9wWDkz7rYwP3caO6s3lnlRBCgvDc04Tvh8X+s6jOoaBK3MNeNkujvUOSkJT8z9VHHYc/BgpOJ/xhqAnqrKaJgwzgAf3PcXT1VNmL2tW/dT/tJXJ7V/w3wgUUIl3+rDZMJ1qePlkrkyvt8IKSFsf/V0Dplj7SHpr7RMPC6k3cr0rUzPndLK0UIF7aSJ7qBMFHn+QVNTlppSxZjkq9OT690jbguo49Y6Clm50JPw0sLG9TTqvfWA36DuHZPrxUXHnqRwkkUe1aM6bQuZcHzfE3/fDcxh63RNzraN49akAT7x6Al8vrdgGJmDnp0gqoD5lIZtQtbNkBfOLkcD0ZEIgQaxZ4BVn9xmrg3JxVm5OfofSXYsBh6dfXQ8MYThZ/dWmBxnnTwXmJhtwoCwmYDMgMNOWfZeIy/dI346RkqGpk=
+#cache:
+#  directories:
+#  - ~/.m2/repository
+#
+#after_success:
+#- ./cd/before-deploy.sh
+#- ./cd/deploy.sh
```

Comment:
I think we should not comment things out, let's just delete it and use git for retrieving it, if needed in the future.

Reply:
removed